Zombie-Powder is now a drink from shakespeare: Stress tested edition

Shadowflame909 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
This time it's a bittersweet drink that'll knock you right out. Whilst also being a delayed shot to make you go to sleep.
Old PR:  #46817
PRs it intends to close: #46803 #46815

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
Zombie-Powder on injection has very similar cycles to chloral hydrate. But it's a faster ranged KO then CH. At least by one-tier.  It's instant via ingestion.

## Why It's Good For The Game
Functions like the new ranged stuns we have in the game, and the melee stuns.
If someone's trying to KO you. They're either going to have to stand beside you and force you to eat a pill, offer you a drugged drink. Or come behind you with a damp rag and ask the simple question "Does this smell like chloroform to you?"
## Changelog
:cl:
balance: Zombie-Powder is now instant upon ingest. Delayed upon touch and injection.
/:cl:

Shadowflame909 committed:
The Final Countdown

This time it's a bittersweet drink that'll knock you right out and a delayed shot to make you go sleepy time

Shadowflame909 commented:
Got great help from Trazelab and the codebase discord

Ty-the-Smonk commented:
Ok so now it’s apart of the bartenders mass of power gamer drinks, sweet

ExcessiveUseOfCobblestone commented on code/modules/reagents/chemistry/reagents/toxin_reagents.dm:
```diff
@@ -162,19 +162,32 @@
+			M.confused += 1
+			M.drowsyness += 1
+		if(5 to INFINITY)
+			M.fakedeath(type)
```
if you set ingested to TRUE here you wouldn't have to constantly handle the switch (although you should probably rename the var to like `fakedeath_active` or something).

5 is still incredibly low though, If you insist on having it down a person why not have an intermediate knockdown effect or stamina damage /w say slurring. That's more interesting and open to counter besides "lol better already be injected with calomel".

Shadowflame909 replied:
It's just 5 less cycles then Chloral Hydrate and it still only produces 2u here per 5u of morphine, carpo, and copper.

ExcessiveUseOfCobblestone replied:
5 less cycles is a pretty big deal, the bonus of ZP should come from the fakedeath and the plausible deniability associated with that (oh romeo my romeo), not really the hardstun. That aspect should be the reward of doing more difficult transfer methods via the ingest.

ExcessiveUseOfCobblestone commented:
you gotta put close (num) for each one

81Denton reviewed (changes requested):
What @ExcessiveUseOfCobblestone said

Shadowflame909 commented:
I'll see to it today.

Shadowflame909 committed:
This is starting to look like druggy alcohol

Added slurring. Bumped up cycles till fake death by 3.

Obvious hard-stun tell at 5. Goes away extremely fast if you don't have enough ZP to KO with. As it's just stamina damage.

Shadowflame909 commented:
Hopefully, that should satisfy all requests. Always tested and it works perfectly.

Gonna need 15u syringes of Zombie Powder if you want to KO with it though.

It purges really fast and 2 units for 15 units of power ratio is still the same

ExcessiveUseOfCobblestone commented:
>>>>>> you gotta put close (num) for each one

ExcessiveUseOfCobblestone commented:
this doesn't metabolize until you're pretend dead, is that intended?

ExcessiveUseOfCobblestone commented on code/modules/reagents/chemistry/reagents/toxin_reagents.dm:
```diff
@@ -162,19 +162,36 @@
+			M.drowsyness += 1
+			M.slurring += 3
+		if(5 to 8)
+			M.adjustStaminaLoss(60, 0)
```
```suggestion
			M.adjustStaminaLoss(40, 0)
```
This means they max at 8 (right before they're 💤) instead of being btfo'd on cycle 7, not including other sources of stamina.

ExcessiveUseOfCobblestone replied:
to clarify, when I suggested knockdown I meant knockdown rather than paralysis.

ExcessiveUseOfCobblestone commented on code/modules/reagents/chemistry/reagents/toxin_reagents.dm:
```diff
@@ -162,19 +162,36 @@
+			M.slurring += 3
+		if(5 to 8)
+			M.adjustStaminaLoss(60, 0)
+		if(8 to INFINITY)
```
```suggestion
		if(9 to INFINITY)
```
double dip

Shadowflame909 committed:
Update code/modules/reagents/chemistry/reagents/toxin_reagents.dm

Co-Authored-By: moo <11748095+ExcessiveUseOfCobblestone@users.noreply.github.com>

Shadowflame909 committed:
Update code/modules/reagents/chemistry/reagents/toxin_reagents.dm

Co-Authored-By: moo <11748095+ExcessiveUseOfCobblestone@users.noreply.github.com>

Shadowflame909 commented:
> 
> 
> this doesn't metabolize until you're pretend dead, is that intended?

Yeah that's totally fine. Since when metabolization starts, it expunges itself really fast. Don't want to oversell it. But It's like using old charcoal on it. Tiny unit value + fast expunging.

Thus making it a valuable knock-out tool. But you're going to need to get moving fast, and use something else/a whole lot more of it. To keep someone down.

ExcessiveUseOfCobblestone commented:
it's not a metab of 0.4? I'm not seeing where you're getting this "rapid expunge" comment from

Shadowflame909 commented:
per 15 units of the stuff to create it. You only get 2 units

it expunges at a rate of 2 units per tick

the whole thing is that with the cycle feature metabing. you wouldn't ever reach the fake-death. Nigh the truth.

It keeps the old balance and I don't get yelled at for completely breaking zombie powder

ExcessiveUseOfCobblestone commented:
where are you getting the 2/iteration from... it's 0.4 metab....

I'm fully aware of the recipe ratio though

Shadowflame909 commented:
I think there's been some sort of of miscommunication.

Yeah the recipe is 2 units per 5 units of all 3 reagents

ExcessiveUseOfCobblestone commented:
>I think there's been some sort of of miscommunication.

correct, here:

>it expunges at a rate of 2 units per tick
